src/arb_fmpz_poly.h

```diff
@@ -46,7 +40,7 @@ ulong arb_fmpz_poly_deflation(const fmpz_poly_t input);
 
 void arb_fmpz_poly_complex_roots(acb_ptr roots, const fmpz_poly_t poly, int flags, slong target_prec);
 
-ARB_FMPZ_POLY_INLINE
+FLINT_FORCE_INLINE
 void arb_fmpz_poly_cos_minpoly(fmpz_poly_t res, ulong n)
```

Comment:
Feel free to just remove this function instead.

Reply:
Done!